### DISCUSSION TITLE
Illogical naming restrictions for pagePath and assetFolders

### DISCUSSION THREAD

**Discussion by rubengmez** [requarks/wiki#5665] [Error / Bug Report] [answered]
The naming restrictions are illogical.
For healthy permissions set, **naming restrictions should be the same for pagePath and assetFolders** 

Example: 
If you have a page with path /Test/test you can't create an assetFolder with name 'Test' to upload the assets because of uppercase restriction.
So you need to set two page rules, one for the page path "Test" and one for read/manage assets of 'test' folder.

**Comment by NGPixel** [comment, marked as answer]
This is already addressed in the upcoming 3.0 release.
